### PR TITLE
Fix the spec file

### PR DESCRIPTION
**Pull request by isuruf**
Checklist
* [ ] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/libgfortran.spec [hunk 1]
```diff
@@ -10,4 +10,3 @@
 *libgcc:
 %{static-libgcc|static:                                                          %{!m32:%:version-compare(>= 10.6 mmacosx-version-min= -lSystem)}                  -lgcc_eh -lgcc;                                                             shared-libgcc|fexceptions|fgnu-runtime:                                          %:version-compare(!> 10.5 mmacosx-version-min= -lSystem)                  %:version-compare(>< 10.5 10.6 mmacosx-version-min= -lSystem)          %:version-compare(!> 10.5 mmacosx-version-min= -lSystem)             %:version-compare(>= 10.5 mmacosx-version-min= -lSystem)                  -lgcc ;                                     :%:version-compare(>< 10.3.9 10.5 mmacosx-version-min= -lSystem)        %:version-compare(>< 10.5 10.6 mmacosx-version-min= -lSystem)          %:version-compare(!> 10.5 mmacosx-version-min= -lSystem)           %:version-compare(>= 10.5 mmacosx-version-min= -lSystem)                  -lgcc }
```
Comment (beckermr) [inline recipe/libgfortran.spec:11]: Does this line need a closing `}`? I have no idea what is supposed to be in a spec file.

Comment (isuruf) [reply recipe/libgfortran.spec:11]: there are 2 opening brackets and 2 closing ones

Comment (beckermr) [reply recipe/libgfortran.spec:11]: ohhhh I missed it! sorry for the noise!

**Comment by beckermr** [comment]
@isuruf I am seeing a lot of this for the test suite

```
2020-09-21T17:11:51.0011420Z + [[ '' == \o\s\x\-\6\4 ]]
```

Are you sure `target_platform` is set for test scripts?

**Comment by beckermr** [comment]
Maybe we should use selectors in the recipe to skip the tests when building the cross compilers or at least use a different test file?

**Comment by beckermr** [comment]
For example, the tests were skipped here but I think they should have been run:

https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=212628&view=logs&j=4b3757f9-99c5-5891-6c0c-93d7030cdf7d&t=f047df50-a233-5ae9-bf8f-c7dd8e5684ad&l=15738

**Pull request merged**